website/docs/r/app_service_managed_certificate.html.markdown

```diff
@@ -0,0 +1,114 @@
+---
+subcategory: "App Service (Web Apps)"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_managed_certificate"
+description: |-
+  Manages an App Service Managed Certificate.
+---
+
+# azurerm_app_service_managed_certificate
+
+Manages a App Service Managed Certificate.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "exampleRG"
+  location = "West Europe"
+}
+
+resource "azurerm_app_service_plan" "example" {
+  name                = "examplePlan"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  kind                = "Linux"
+  reserved            = true
+
+  sku {
+    tier = "Basic"
+    size = "B1"
+  }
+}
+
+resource "azurerm_app_service" "example" {
+  name                = "exampleApp"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  app_service_plan_id = azurerm_app_service_plan.example.id
+}
+
+resource "azurerm_dns_txt_record" "test" {
+  name                = join(".", ["asuid", "%s"])
+  zone_name           = data.azurerm_dns_zone.test.name
+  resource_group_name = data.azurerm_dns_zone.test.resource_group_name
+  ttl                 = 300
+
+  record {
+    value = azurerm_app_service.test.custom_domain_verification_id
+  }
+}
+
+resource "azurerm_app_service_custom_hostname_binding" "example" {
+  hostname            = "mycustomhost.contoso.com"
```

Comment:
this may as well become a reference to the one above to enforce the dependency, I think?

```suggestion
  hostname            = azurerm_dns_txt_record.test.name
```

Reply:
Not quite - both records are required but the TXT is for auth, the CNAME + subdomain is required here, so perhaps: 
```suggestion
  hostname            = join(".", [azurerm_dns_cname_record.example.name, azurerm_dns_cname_record.example.zone_name])
```
to show the dependency on both?